rustls/Cargo.toml

```diff
@@ -18,6 +18,7 @@ rustversion = { version = "1.0.6", optional = true }
 [dependencies]
 aws-lc-rs = { version = "1.6", optional = true, default-features = false, features = ["aws-lc-sys"] }
 log = { version = "0.4.4", optional = true }
+once_cell = { version = "1" }
```

Comment:
Style: just `once_cell = "1"` would suffice.

Maybe we should make a note somewhere that, once we bump the MSRV to 1.70, we could consider using `std::sync::OnceLock` instead of this extra dependency.

Reply:
Added a comment next to this line about the MSRV.